.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,41 @@
-**Please check if the PR fulfills these requirements**
-- [ ] The commit message follows our guidelines: TBD
-- [ ] Tests for the changes have been added (for bug fixes / features)
-- [ ] Docs have been added / updated (for bug fixes / features)
+<!--- Provide a general summary of your changes in the Title above -->
 
+## Description
 
-**What kind of change does this PR introduce?** (check one with "x")
-```
-[ ] Bugfix
-[ ] Feature
-[ ] Code style update (formatting, local variables)
-[ ] Refactoring (no functional changes, no api changes)
-[ ] Build related changes
-[ ] CI related changes
-[ ] Other... Please describe:
-```
+Describe your changes in detail
 
-**What is the current behavior?** (You can also link to an open issue here)
+## Motivation and Context
 
+Why is this change required? What problem does it solve?
+If it fixes an open issue, please link to the issue here.
 
+## How Has This Been Tested?
 
-**What is the new behavior?**
+Please describe in detail how you tested your changes.
 
+Include details of your testing environment, and the tests you ran to
 
-**Other information**:
+see how your change affects other areas of the code, etc.
```

Comment:
Does that sentence is `Say how your change...` ? or is it `see how your change...`.